src/components/admin/applications/AllApplicationsTable.tsx

```diff
@@ -66,19 +88,143 @@ const AllApplicationsTable: React.FC = () => {
   if (error) {
     return <ErrorScreen error={error} />;
   }
+  if (error2) {
+    return <ErrorScreen error={error2} />;
+  }
+
+  // Filters
+
+  interface GroupOption extends OptionBase {
+    label: string;
+    value: string;
+  }
+
+  const statusOptions = [
+    {
+      label: "Draft",
+      value: "DRAFT",
+    },
+    {
+      label: "Applied",
+      value: "APPLIED",
+    },
+    {
+      label: "Accepted",
+      value: "ACCEPTED",
+    },
+    {
+      label: "Waitlisted",
+      value: "WAITLISTED",
+    },
+    {
+      label: "Confirmed",
+      value: "CONFIRMED",
+    },
+    {
+      label: "Denied",
+      value: "DENIED",
+    },
+  ];
+
+  const applicationBranchOptions: any[] = [];
+  const confirmationBranchOptions: any[] = [];
+
+  branches &&
+    branches
```

Comment:
instead of coding it like a push, you can just do like `const applicationBranchOptions = branches ? branches.filter.map...... : []` for both of these. itll make the code a lot more readable